### ISSUE TITLE
Open Weather Map API calls not working

### ISSUE THREAD

**Issue by cdlenfert** [mattrossman/forecaswatch2#68]
I recently decided to try to switch the weather service in Forecaswatch 2 over to OpenWeatherMap. I set up an account and found my API key. I verified my email address in the account and read in a confirmation email that my API key would be active in a couple of hours. If I make an HTTP request in my browser (example below) I get relevant data back. 

```
https://api.openweathermap.org/data/2.5/weather?lat={lat}&lon={lon}&appid=[{API key}](https://home.openweathermap.org/api_keys)
```
Is there anything I am missing or that can be done to get this awesome watchface to pull weather data again?

**Comment by mattrossman** [comment]
The OpenWeatherMap provider is working in the watchface my end, maybe the new account just needs more time to fully settle?

The [provider implementation](https://github.com/mattrossman/forecaswatch2/blob/a84a3a79753fe6dddfa6d066b8b1db4df7c20fc7/src/pkjs/weather/openweathermap.js) uses the `/onecall` endpoint with a couple other params like:

```
https://api.openweathermap.org/data/2.5/onecall?lat={LAT}&lon={LON]&appid={API_KEY}&units=imperial&exclude=alerts,minutely
```

Maybe try that request format and see if you have any issues. Otherwise I'd wait and try the watchface again later.

**Comment by cdlenfert** [comment]
@mattrossman thank you for the quick response. I actually looked through the same cod you linked and noticed that the API endpoint is `onecall` and found that when I try to hit that endpoint on an Open Weather Map free plan my API key was not allowed to access it. I subscribed to the "One Call by Call" plan (which requires a credit card to be on file, but allows for 1,000 free API calls a day, and allows you to set a hard limit so you never get charged). Once I made this plan change (and cleared my browser cache), the API calls to `https://api.openweathermap.org/data/2.5/onecall?lat={LAT}&lon={LON]&appid={API_KEY}&units=imperial&exclude=alerts,minutely` started working for me and the watchface also started working again!
I'm all set now. Thanks again for your help!
